build.proj

```diff
@@ -65,4 +65,5 @@
 
   <Import Project="build/Microsoft.DotNet.Cli.Prepare.targets" />
   <Import Project="build/Microsoft.DotNet.Cli.Package.targets" />
+  <Import Project="build/Microsoft.DotNet.Cli.Compile.targets" />
```

Comment:
probably not important, but my OCD kicks in when these are out of order.  Can "Compile" be imported before "Package"?